clients/js/cmds/sui.ts

```diff
@@ -37,6 +63,28 @@ exports.command = 'sui';
 exports.desc = 'Sui utilities ';
 exports.builder = function(y: typeof yargs) {
   return y
+    .command("get-owned-objects", "Get owned objects by owner", (yargs)=>{
+      return yargs
+        .option("network", network_options)
+        .option("rpc", rpc_description)
+        .option("owner", {
+          alias: "o",
+          describe: "Owner address",
+          required: true,
+          type: "string",
+        })
+    }, async (argv)=>{
+      const network = argv.network.toUpperCase()
```

Comment:
there is a nice util that @kcsongor wrote for the Aptos CLI that asserts the network, can you move that to a shared util file and use it here too? https://github.com/wormhole-foundation/wormhole/blob/4c7df41984b6f4947773846050bdd68efc3a8cff/clients/js/cmds/aptos.ts#L47